pkgs/by-name/un/unicodeit/package.nix

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
is this standard practice? otherwise you could also add the alias to the appropriate file in top-level